### ISSUE TITLE
jdk21 release

### ISSUE THREAD

**Issue by delanym** [tcurdt/jdependency#198]
hi. Can we have another release for Java 21?

**Comment by tcurdt** [comment]
Happy to release. But I guess this needs some testing first.
Have your tried or have run into problems?

The current CI only tests up to 19.

https://github.com/tcurdt/jdependency/blob/master/.github/workflows/ci.yml

I would also need to check if this needs an ASM upgrade.

**Comment by delanym** [comment]
I cloned and set my project shade-plugin to use the latest. No problems.

But I'm confused about something. Your pom already specifies asm 9.5 since 4 months ago
https://github.com/tcurdt/jdependency/commit/09234114ab1316c3d9916ce65267be6211f4b27d

So what's with the 5 open pull requests making the same change?

**Comment by tcurdt** [comment]
> So what's with the 5 open pull requests making the same change?

That's a very good question. They don't make much sense and I also didn't notice them. It's a bit weird given the release in August. Anyway, I closed them and turned on java 21 for testing.

Unfortunately there is now support for java 21 on github yet.

https://github.com/tcurdt/jdependency/actions/runs/6220534843

> Could not find satisfied version for semver 21. 
> Available versions: 20.0.2+9,

I have added java 20 to the testing matrix at least.
